tests/quickchecks.rs

```diff
@@ -1,3 +1,4 @@
+#![cfg(not(miri))]
```

Comment:
Can you add a comment describing why these tests are disabled when running under miri?

Nitpick: newline between this attribute and the `use` statements, please.

```suggestion
#![cfg(not(miri))]

```